sklearn/cross_decomposition/_pls.py

```diff
@@ -584,6 +584,18 @@ class PLSRegression(_PLS):
     y_rotations_ : array, [q, n_components]
         Y block to latents rotations.
 
+    x_mean_ : array, [p]
```

Comment:
Let's correct all the syntaxes to follow the documentation guide. We can restrict to the `Attributes` section.

We should use the following:

```suggestion
    x_mean_ : ndarray of shape (p,)
```

and in case of a 2D array

```python
X_scores_ : ndarray of shape (n_sammples, n_components)
```

Reply:
Thank you very much for your review comment. 
I fixed all parameters of `PLSRegression` to follow the documentation guide.